src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -79,6 +80,13 @@ public string[] Verbs
             }
         }
 
+        [CLSCompliant(false)]
+        public SecureString Password
```

Comment:
I think that it the only code that uses `Password`. I'm working on bring it back. Thanks for the heads up.